frontend/src/index.tsx

```diff
@@ -83,6 +84,16 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
                     </SidebarWrapper>
                   </ProtectedRoute>
                 ),
+                children: [
+                  {
+                    path: AppRoute.CHATS_$ID,
+                    element: (
+                      <ProtectedRoute>
+                        <ChatMessages />
+                      </ProtectedRoute>
+                    ),
+                  },
+                ],
```

Comment:
We should do something similar to sign-in/sign-up flow, use one component and in it detect by parameters is it a new chat or existed one
```suggestion
                children: [
                  {
                    path: AppRoute.CHATS_$ID,
                    element: (
                      <ProtectedRoute>
                        <Chat />
                      </ProtectedRoute>
                    ),
                  },
                ],
```

Reply:
but why we can't simply use outlet to render messages?